templates/4_Post_Clustering.ipynb

```diff
@@ -6,9 +6,10 @@
    "metadata": {},
```

Comment:
<p>Is there a better way to do this other than raw printing everything out? At the very least, I think this should be formatted so it lists a separate cluster name per row.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1095/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
Not really an automated way to do this since the user needs to manually define the cluster grouping dictionary. I'll fix it so it prints like below.
```
0              CD4T
1              CD8T
2     CD14_monocyte
3             Bcell
4             other
5     M2_macrophage
6      immune_other
7     M1_macrophage
8               APC
9            stroma
10      endothelium
11    Myofibroblast
12       tumor_ck17
13       tumor_ecad
Name: cell_meta_cluster, dtype: object
```